### ISSUE TITLE
Support for PW from file

### ISSUE THREAD

**Issue by jairov4** [jazzdd86/phpVirtualbox#12]
Security risk using password as env variable.
Please add the capability to use something like SRV1_PW_FILE=/password_file.txt in servers-from-env.php
It would make possible use docker secrets with phpVirtuabox in compose or swarms with better security.

**Comment by jazzdd86** [comment]
Good point, I will need some time to make the changes. Feel free to do a merge request for a faster solution to your request.
